taigit/src/actions/taigaActions.js

```diff
@@ -18,7 +19,10 @@ export const grabTaigaData = (infoForApiCall) => dispatch => {
    *   );
    * 
    */
-  dispatch({type: GRAB_TAIGA_DATA, payload:'mockApiData'})
+  project_info('sanaydevi-ser-574') // give the slug name
```

Comment:
I would change the hardcoded slug to a variable passed inplace of _infoForApiCall_ so it would be like:
```
export const grabTaigaData = (slugName) => dispatch => {
...
project_info(slugName)
...
```

Reply:
Yeah. This is not the part of this task as well, I can revert this.